pallets/bootstrap/src/lib.rs

```diff
@@ -157,6 +157,9 @@ pub mod pallet {
 
 		type PoolCreateApi: PoolCreateApi<AccountId = Self::AccountId>;
 
+		#[pallet::constant]
+		type BootstrapUpdateBuffer: Get<Self::BlockNumber>;
```

Comment:
```suggestion
		type BootstrapUpdatePeriod: Get<Self::BlockNumber>;
```

i think period would be a better world ?

Reply:
Period would indicate that the bootstrap is modifiable within 300 blocks before start, whereas it is the otherway.